src/inference_profiling/cloud_run_logs.txt

```diff
@@ -0,0 +1,7 @@
+Memory utilization was almost always ~50%
+
+CPU performance 53% for 10 requests and around 80% for 1000 requests
+
+Container count 30 for 10 requests and around 139 for 1000 requests
```

Comment:
Why would there be 30 containers for 10 requests? Seems counter intuitive

Reply:
It's an approximate from the observed value from the data. Will perform cross-verification again.

Reply:
Yes, there are 10 active containers for 10 requests